### ISSUE TITLE
while training - loss parameter almost doesnt decrease

### ISSUE THREAD

**Issue by tamirm2009** [thtrieu/darkflow#723]
Hi 
I'm running training process but loss parameter doesnt decreasing:
first step it was 107% loss. On Step 425 loss is 102% only. What Should I do ?
Is something suspicious/wrong ? 
I have 6 classes 1,2,3,4,5,6 . And dataset of 59 images only(XML file for each image).
I'm training using yolov2-tiny-voc file --> edited to yolov2-tiny-voc-6c . Weights -> yolov2-tiny.weights(43896KB).
BatchSize/Subdivision is relevant?
[net]
/# Testing
batch=16
subdivisions=4
/# Training
/# batch=64
/# subdivisions=2

I added / after the # because its bolding in github the text afterwards.(So my cfg file is without /).

cmdline run string:
python flow --model cfg/yolov2-tiny-6c.cfg --load bin/yolov2-tiny.weights --train --annotation new_model_data/annotations --dataset new_model_data/images

![annotation_xml_for_example](https://user-images.githubusercontent.com/38405527/38805438-78ab70f8-417e-11e8-8197-9edc29c91f0b.JPG)
![images_folder](https://user-images.githubusercontent.com/38405527/38805440-78cf7d36-417e-11e8-8a76-63070e64e848.JPG)
![annotation_folder](https://user-images.githubusercontent.com/38405527/38805442-78f40782-417e-11e8-8a51-c4f3baf6ce3f.JPG)
![image](https://user-images.githubusercontent.com/38405527/38805303-07ea747c-417e-11e8-937a-0e4ea931452e.png
[yolov2-tiny-voc-6c.txt](https://github.com/thtrieu/darkflow/files/1914940/yolov2-tiny-voc-6c.txt)

**Comment by LukasRondelez** [comment]
Hi,
I noticed having the same issue when training tiny-yolo on 1 class on my own dataset. Adjusting the learning rate (added --lr 0.1 just to check the effect), made the avg loss drop faster. 
It dropt from 105 to 8.9 in 66 steps. Next steps I'm getting NaN. Will be looking in to that later

**Comment by ilgindogan** [comment]
Hello, I am using tiny yolov2 algorithm with my custom object recognition and tracking. But ı have one problem when ı add new class on labels.txt at the same time added new photo my  images path and xml path. When accuracy of classes is hardly decreased. İmages of all classes is various like first class has 1.450 photo and second class 473 photo etc... All classes have different photo. Due to This situation  would I encounter an error.? Because  ı have error about when class number increased my accuracy of each classes is decreased as hardly. such as %50-60 threshold will be down %5-10. Can you help to me about this subject. What can I do ? Thank you.
